packages/fxa-content-server/app/scripts/models/auth_brokers/oauth-webchannel-v1.js

```diff
@@ -72,6 +72,7 @@ const OAuthWebChannelBroker = OAuthRedirectAuthenticationBroker.extend({
       });
       return this.set('chooseWhatToSyncWebV1Engines', syncEngines);
     }
+    return proto.onFxaStatus.call(this, response);
```

Comment:
This is a tangential change, I noticed none of the `onFxaStatus` were calling the super class method, which it should so the base class can set the [supportsPairing](https://github.com/mozilla/fxa/blob/88f6119cc75ae75bd26f87cf5e42dc71e95411d4/packages/fxa-content-server/app/scripts/models/auth_brokers/base.js#L69) capability.